packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md

```diff
@@ -66,15 +60,15 @@ Tokens expire automatically after a certain period and can also be deactivated a
 
 ### Token expiration
 
-Tokens are valid for 30 days from creation or last use, so that the 30 day expiration automatically refreshes with each API call. Tokens that aren't used for 30 days expire. The 30-day period is currently fixed and can't be changed for your organization.
+Tokens are valid for 30 days from creation or last use, so that the 30 day expiration automatically refreshes with each API call. Tokens that aren't used for 30 days expire. The 30-day period is fixed and can't be changed for your organization.
```

Comment:
```suggestion
Tokens are valid for 30 days from creation or last use, so that the 30 day expiration automatically refreshes with each API call. Tokens that aren't used for 30 days expire. The 30-day period is fixed and can't be changed for your org.
```